src/client/Microsoft.Identity.Client.Brokers/RuntimeBroker.cs

```diff
@@ -108,6 +108,16 @@ internal class RuntimeBroker : IBroker
                     _logger.Verbose("[WamBroker] Account information was not provided. Using an account picker.");
                 }
 
+                // if PopAuthenticationConfiguration is set, proof of possesion will be performed via the runtime broker
+                if (authenticationRequestParameters.PopAuthenticationConfiguration != null)
```

Comment:
There are a bunch of other places like AcquireTokenSilent where POP needs to be set.